src/terminal_chat_console.cpp

```diff
@@ -333,10 +333,6 @@ void TerminalChatConsole::step(int ch)
 			continue;
 
 		std::wstring error_message = utf8_to_wide(Logger::getLevelLabel(p.first));
-		if (!g_settings->getBool("disable_escape_sequences")) {
```

Comment:
it was removed in the sense that it's always false, so this code should be kept.
however I already added a fix to #15839 so we can also just close this one.

Reply:
I like it. I'll close this.